Automattic/wp-calypso
# [WIP] Command Palette: Prepare component to be consumed externally

## Proposed Changes

Refactorizes the Command Palette component so it can be consumed outside of Calypso:
- Usages of the internal Calypso Redux state have been replaced with custom query hooks implemented within the Command Palette component.
- Usages of internal components not exposed as public packages have been replaced with custom components implemented within the Command Palette component.

Note that the `useCommandsArrayWpcom` has not been yet refactorized.

Once the component is fully decoupled of Calypso internal state/component, next step would be to move it to the `
apps` folder so it can be built as an independent app that be loaded either in Calypso or in WP Admin.

## Testing Instructions

- Use the Calypso live link below
- Open the command palette
- Make sure there are no regressions
